Fabric.Authorization.API/Modules/IdentitySearchModule.cs

```diff
@@ -22,7 +24,7 @@ public class IdentitySearchModule : SearchModule<IdentitySearchRequest>
         {
             _identitySearchService = identitySearchService;
 
-            Get("/", async _ => await SearchIdentities().ConfigureAwait(false), null, "GetGroup");
+            Get("/", async _ => await SearchIdentities().ConfigureAwait(false), null, "SearchIdentities");
         }
 
         private async Task<dynamic> SearchIdentities()
```

Comment:
We probably talked about this before, I don't recall at this point, but should we be mixing identity based scopes in authorization? I can see the potential need to secure the call to Fabric.Identity, but perhaps the scope that the client application uses is an authorization based scope that is different from fabric/authorization.read.

Reply:
I agree. I was thinking that authorization would use the identity.read scope to get the users from identity, but the client calling authorization would use either authorization.read or a new authorization scope